### PR TITLE
Add config check for alluxio.master.ttl.checker.interval at start time

### PR DESCRIPTION
**Pull request by MEILIDEKCL**
### What changes are proposed in this pull request?

#17405

### Why are the changes needed?
add pre-check for alluxio.master.ttl.checker.interval, to avoid resources waste when starting alluxio.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

File: core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java [hunk 1]
```diff
@@ -675,6 +675,7 @@ public void userFileBufferBytesOverFlowException() {
   public void shortMasterHeartBeatTimeout() {
     mConfiguration.set(PropertyKey.MASTER_STANDBY_HEARTBEAT_INTERVAL, "5min");
     mConfiguration.set(PropertyKey.MASTER_HEARTBEAT_TIMEOUT, "4min");
+    mConfiguration.set(PropertyKey.MASTER_TTL_CHECKER_INTERVAL_MS, "0");
```
Comment (jiacheliu3) [inline core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java:678]: please add a separate UT for your case. This UT is testing for `shortMasterHeartBeatTimeout`.

File: core/common/src/main/java/alluxio/conf/InstancedConfiguration.java [hunk 2]
```diff
@@ -481,6 +482,17 @@ private void checkWorkerPorts() {
...
+  
+  /**
+   * Checks that master TTL checker interval
+   * @throws IllegalStateException if the TTL check interval <= 0
+   */
+  private void checkMasterTTLInterval() {
+    long interval = getMs(PropertyKey.MASTER_TTL_CHECKER_INTERVAL_MS);
+    checkState(interval > 0, 
+        "invalid value of alluxio.master.ttl.checker.interval %s, "
+        + "it must be lager than 0", interval);
```
Comment (jiacheliu3) [inline core/common/src/main/java/alluxio/conf/InstancedConfiguration.java:494]:
```suggestion
        "invalid value of " + PropertyKey.MASTER_TTL_CHECKER_INTERVAL_MS + "=%s, "
        + "it must be greater than 0", interval);
```

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]
Added a few comments. Thanks for the improvement!

File: core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java (hunk 1 above)
Comment (MEILIDEKCL) [reply core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java:678]: yes, thank you.

**Review by jiacheliu3 — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
why is this merging into `branch-2.9`? Should this be `master-2.x` instead?

**Comment by MEILIDEKCL** [comment]
> why is this merging into `branch-2.9`? Should this be `master-2.x` instead?

Because I just use branch-2.9, and I think in other branches, the problem still exists, so this should be merged into master-2.x

**Comment by jiacheliu3** [comment]
For an existing PR, you can't change the destination. So could you create another PR to `master-2.x` using the same code? Thanks!

**Comment by MEILIDEKCL** [comment]
> For an existing PR, you can't change the destination. So could you create another PR to `master-2.x` using the same code? Thanks!

Yes, of course. Thank you for reviewing it.

**Comment by MEILIDEKCL** [comment]
Hello, I have created a new pr, Thank you.

**Pull request closed without merging**